test/account/UpgradeableModularAccount.t.sol

```diff
@@ -42,6 +42,11 @@ contract UpgradeableModularAccountTest is AccountTestBase {
     event PluginUninstalled(address indexed plugin, bool indexed callbacksSucceeded);
     event ReceivedCall(bytes msgData, uint256 msgValue);
 
+    // helper function to compress 2 gas values into a single bytes32
+    function _encodeGas(uint256 g1, uint256 g2) internal pure returns (bytes32) {
+        return bytes32(uint256(g1 << 128 + uint128(g2)));
+    }
+
```

Comment:
Could we move this helper function to `AccountTestBase` so we don't have to define it multiple times?